src/components/common/ActionSheet/styles.android.js

```diff
@@ -0,0 +1,46 @@
+import { StyleSheet } from 'react-native';
+
+import { Colors } from '@/constants/theme';
+
+const ITEM_RADIUS = 14;
+
+export default StyleSheet.create({
+  modal: {
+    backgroundColor: Colors.Black.Primary,
+  },
+  topContainer: {
+    backgroundColor: Colors.Black.Primary,
+    borderRadius: ITEM_RADIUS,
+    margin: 8,
+    marginBottom: 0,
+  },
+  titleContainer: {
+    paddingVertical: 12,
+    paddingHorizontal: 8,
+  },
+  title: {
+    color: Colors.White.Secondary,
+  },
+  subtitle: {
+    fontSize: 12,
+    color: Colors.White.Secondary,
+  },
+  item: {
+    paddingHorizontal: 8,
+    height: 60,
+    justifyContent: 'center',
+  },
+  itemBorder: {},
```

Comment:
delete this

Reply:
left it because in the future it will cause a Typescript error (when we migrate all)